config/application.rb

```diff
@@ -19,7 +19,7 @@ class Application < Rails::Application
 
     # This needs to be set for correct images URLs in emails
     # DON'T FORGET to ALSO set this in `config/initializers/carrierwave.rb`
-    # config.action_mailer.asset_host = "https://participer.loire-atlantique.fr"
+    config.action_mailer.asset_host = "https://participer.loire-atlantique.fr"
```

Comment:
To ensure homogeneity between `config/initializers/carrierwave.rb` and this file we should add a guard clause like following: 

```suggestion
    config.action_mailer.asset_host = "https://participer.loire-atlantique.fr" if Rails.env.production?
```